spas-bozhanov/homework1/CV/main-homework.css

```diff
@@ -0,0 +1,256 @@
+	background: linear-gradient(to right,  #f2f6f8 0%,#d8e1e7 11%,#b5c6d0 77%,#e0eff9 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#f2f6f8', endColorstr='#e0eff9',GradientType=1 ); /* IE6-9 */	
+
+	border: 1px solid black;
+	height: auto;
+	margin-left: 220px;
+	margin-bottom: 10px;
+
+	-webkit-box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+	-moz-box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+	box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+}
+
+#education-page main>section {
+	background: #f2f6f8; /* Old browsers */
+	background: -moz-linear-gradient(left,  #f2f6f8 0%, #d8e1e7 11%, #b5c6d0 77%, #e0eff9 100%); /* FF3.6-15 */
+	background: -webkit-linear-gradient(left,  #f2f6f8 0%,#d8e1e7 11%,#b5c6d0 77%,#e0eff9 100%); /* Chrome10-25,Safari5.1-6 */
+	background: linear-gradient(to right,  #f2f6f8 0%,#d8e1e7 11%,#b5c6d0 77%,#e0eff9 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#f2f6f8', endColorstr='#e0eff9',GradientType=1 ); /* IE6-9 */	
+
+	border: 1px solid black;
+	height: auto;
+	margin-left: 220px;
+	margin-bottom: 10px;
+
+	-webkit-box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+	-moz-box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+	box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+}
+
+#personal-skills-page main>section {
+	background: #f2f6f8; /* Old browsers */
+	background: -moz-linear-gradient(left,  #f2f6f8 0%, #d8e1e7 11%, #b5c6d0 77%, #e0eff9 100%); /* FF3.6-15 */
+	background: -webkit-linear-gradient(left,  #f2f6f8 0%,#d8e1e7 11%,#b5c6d0 77%,#e0eff9 100%); /* Chrome10-25,Safari5.1-6 */
+	background: linear-gradient(to right,  #f2f6f8 0%,#d8e1e7 11%,#b5c6d0 77%,#e0eff9 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#f2f6f8', endColorstr='#e0eff9',GradientType=1 ); /* IE6-9 */	
+
+	border: 1px solid black;
+	height: auto;
+	margin-left: 220px;
+
+	-webkit-box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+	-moz-box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+	box-shadow: 10px 10px 8px 0px rgba(0,0,0,0.75);
+}
+
+#main-page h1 {
+	padding-top: 3em;
+	text-align: center;
+	font-size: 2em;
+	letter-spacing: 3px;
+}
+
+#main-page main>section>p {
+	text-align: center;
+	font-size: 1.5em;
+}
+
+dl:hover {
+	background-color: grey;
+	color: white;
+}
+
+dt {
+	padding-left: 1em; 
+	font-weight: bold;
+}
+
+dd {
+	padding-left: 3em;
+}
+
+button {
+	margin-left: 0.3em;
+}
+
+footer>section {
+	color: lightgrey;
+}
+
+#form-date {
+		font-family: arial, verdana, sans-serif;
+	}
+
+#form-position {
+	width: 40%;
+}
+
+#form-country {
+	width: 40%;
+}
+
+#form-employer {
+	width: 40%;
+}
+
+#form-education {
+	width: 40%;
+}
+
+#form-about {
+	width: 40%;
+}
+
+#form-business {
```

Comment:
@spas-bozhanov 
Since you have the same style for all elements you can use the "," combinator to combine all selectors.